### DISCUSSION TITLE
Need help editing some properties - New to editing themes

### DISCUSSION THREAD

**Discussion by F1mmel** [nptr/msstyleEditor#160] [General]
Hello,

I'm still pretty new when it comes to changing themes. 
I use msstyleEditor to load a theme, then modify it and then save and apply it. I've managed to do that, but I have a few questions about certain properties that I can't find.

1. I want to change the small thin lines between the left and center areas. So where the scrollbar appears.
(outlined in red)
![image](https://github.com/user-attachments/assets/0d99ad6b-d4a5-4c64-8a7f-2f238b9dbdf9)

2. I would also like to know the exact property that I need to change because I want to change the hover color when hovering over a folder or file in Explorer. I think that should be able to be changed under "Symetrics".
![image](https://github.com/user-attachments/assets/3e319b53-9e8e-4ba6-aa55-fed3c1cc73aa)
Top: Current selected
Bottom: Hovered

I hope someone can help me as I'm still fairly new.

**Comment by Undisputed00x** [comment]
For 1. you can only edit the seperator between navigation pane and DUI view, as for the other above are Xaml elements and can't be modified by the msstyles.

1. I think its `PreviewPane > NavPaneSizer` image
2.  `DarkMode_ItemsView::ListView > ListItem` images in selected and hot states

**Reply by F1mmel** [reply]
Thanks worked like a charm.
I might ask for some things again :)
